Gemfile.lock

```diff
@@ -467,6 +476,7 @@ GEM
 
 PLATFORMS
   ruby
+  x86_64-linux
```

Comment:
I would prefer to not have this in here - I don't actually know why it happens, but what I do know if adding specific platforms seems to break stuff... :(